README.md

```diff
@@ -18,22 +18,22 @@ To begin working on your project, follow these steps:
 1. Clone this repository to your local machine:
 
    ```bash
-   git clone https://github.com/MakeItMVP/LaunchAcademyStarter.git
+   git clone https://github.com/makeitMVPadmin/Content.git
```

Comment:
Since we are the team of ***Data*** not ***Content***, I kindly advise you to change all your changes with ***Content*** to ***Data***.